backend/tests/functional_tests/test_adding_new_label.py

```diff
@@ -57,3 +57,50 @@ def test_add_brush_label(prepare_environment: Any, synchronous_celery: Any) -> N
     label_element_id = json_response['elements'][0]['label_element_id']
     brush_label_element = BrushLabelElement.get(id=label_element_id)
     assert brush_label_element.image
+
+
+def test_add_point_label(prepare_environment: Any, synchronous_celery: Any) -> None:
+    """Test for adding a Label made with Point tool."""
+    api_client = get_api_client()
+    user_token = get_token_for_logged_in_user('admin')
+
+    # Step 1. Add Scan to the system
+    payload = {'category': 'KIDNEYS', 'number_of_slices': 3}
+    response = api_client.post('/api/v1/scans/', data=json.dumps(payload),
+                               headers=get_headers(token=user_token, json=True))
+    assert response.status_code == 201
+    json_response = json.loads(response.data)
+    scan_id = json_response['scan_id']
+
+    # Step 2. Label it with Brush
+    create_tag_and_assign_to_category('EXAMPLE_TAG', 'Example tag', 'KIDNEYS')
+    payload = {
+        'elements': [{
+            'slice_index': 0,
+            'x': 0.25,
+            'y': 0.5,
+            'tag': 'EXAMPLE_TAG',
+            'tool': 'POINT',
+        }],
+        'labeling_time': 12.34,
+    }
+    data = {
+        'label': json.dumps(payload),
+    }
+    response = api_client.post('/api/v1/scans/{}/label'.format(scan_id), data=data,
+                               headers=get_headers(token=user_token, multipart=True))
+    assert response.status_code == 201
+    json_response = json.loads(response.data)
+    assert isinstance(json_response, dict)
+    label_id = json_response['label_id']
+    assert isinstance(label_id, str)
+    assert len(label_id) >= 1
+
+    # Step 3. Fetch details about above Label
+    response = api_client.get('/api/v1/labels/' + label_id, headers=get_headers(token=user_token))
+    assert response.status_code == 200
+    json_response = json.loads(response.data)
+    assert isinstance(json_response, dict)
+    assert len(json_response['elements']) == 1
+    assert json_response['elements'][0]['x'] == 0.25
```

Comment:
It's really strange that this is 0.25 as DB column is Integer... Any thoughts how does it work? o.O

**EDIT:** Ohhhh... I know why! Alembic migration is fine - it created a DB model properly, so it works well. But... there is one test that checks if SQLAlchemy model and DB model are consistent. Why does it not work properly? Hmmm...

Here you can find the test that I'm talking about: https://github.com/jpowie01/MedTagger/blob/master/backend/tests/functional_tests/test_alembic_migrations.py#L14

Would you like to debug it? Maybe above test is wrong?

Reply:
Please, see #304 to find out what was wrong in this test.